torch/_inductor/codegen/triton.py

```diff
@@ -3522,9 +3522,13 @@ def define_kernel(self, src_code, node_schedule):
 
         return kernel_name
 
-    def codegen_template(self, template_node, epilogue_nodes):
+    def codegen_template(
+        self, template_node, epilogue_nodes, only_gen_src_code=False
+    ) -> Optional[str]:
         """
         Codegen a triton template
+
+        If `only_gen_src_code` the src code will be returned instead of codegen'd into the wrapper
         """
         _, (numel, rnumel) = template_node.group
         assert rnumel == 1
```

Comment:
mark_run will add memory allocation lines to the wrapper, we should skip it when we are do benchmarking.

Reply:
Not sure if it's GH bug, I can not attach this comment to line 3538 where `node.mark_run()` is called.

Reply:
Good catch - addressed and extended test so it would have caught